requirements/dev.txt

```diff
@@ -4,10 +4,10 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+-e git+https://github.com/kytos-ng/kytos.git@feature/msg_out_pqueue#egg=kytos
```

Comment:
This commit 768acad9ec41325b2524ab85aee50313fb5ef843 will be reverted before getting merged, I'm using this to make sure the tests passes until the other PRs that this PR depends on get merged.

Reply:
I've just reverted it on [0304666](https://github.com/kytos-ng/flow_manager/pull/88/commits/0304666762ca0c22b45d67793b053670105b9711)